test/gtest/ucs/test_stats.cc

```diff
@@ -253,6 +258,74 @@ class stats_on_exit_test : public stats_file_test {
     }
 };
 
+class stats_aggregate_sum_test : public stats_test {
+public:
+    virtual void init()
+    {
+        ucs_status_t status = ucs_stats_server_start(0, &m_server);
+        ASSERT_UCS_OK(status);
+        stats_test::init();
+    }
+
+    virtual void cleanup()
+    {
+        stats_test::cleanup();
+        ucs_stats_server_destroy(m_server);
+    }
+
+    virtual std::string stats_dest_config()
+    {
+        int port = ucs_stats_server_get_port(m_server);
+        EXPECT_GT(port, 0);
+        return "udp:localhost:" + ucs::to_string(port);
+    }
+
+    virtual std::string stats_trigger_config()
+    {
+        return "timer:0.1s";
+    }
+
+    void
+    read_and_check_aggrgt_sum_stats(ucs_stats_node_t *data_nodes[NUM_DATA_NODES])
+    {
+        const ucs_stats_aggrgt_sum_counter_name_t *aggregate_cnt_names_db = NULL;
+        const ucs_stats_aggrgt_sum_clsid_to_name_db_t *clsid_to_name_db = NULL;
+        ucs_stats_counter_t ucs_stats_aggregate_sum_counters
+                [UCS_STATS_NUM_AGGREGATE_SUM_COUNTERS_MAX];
+        size_t size;
+        size_t n_counters;
+
+        /* Test */
+        ucs_stats_aggregate_sum(ucs_stats_aggregate_sum_counters,
+                                UCS_STATS_NUM_AGGREGATE_SUM_COUNTERS_MAX,
+                                &n_counters, &clsid_to_name_db);
+
+        ucs_stats_aggregate_sum_names_db_get(&aggregate_cnt_names_db, &size);
+
+        ASSERT_NE(0ul, n_counters);
+        ASSERT_FALSE(aggregate_cnt_names_db == NULL);
+
+        /*
+           //Example for processing the statistics
+        uint32_t i;
+
+        for (i = 0; i < n_counters; i++) {
+            if (aggregate_cnt_names_db[i].name && aggregate_cnt_names_db[i].class_name) {
+               printf("[%u] pid=%d | CNT: cls_name=<%s> Name=%s, value=%lu\n",
+                      i,
+                      getpid(),
+                      aggregate_cnt_names_db[i].class_name,
+                      aggregate_cnt_names_db[i].name,
+                      ucs_stats_aggregate_sum_counters[i]);
+           }
+       }
+       */
```

Comment:
???

Reply:
pls remove commented out code

Reply:
This code is an example of how to use the counters. I updated the code and removed the comments.
Please review the updated code.